src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataServiceTest.java

```diff
@@ -372,6 +370,26 @@ public void shouldGetCaseOnlyFilteredAdditionalData() throws ApplicationExceptio
         assertThat(result.getAdditionalFields().stream().noneMatch(f -> f.getLabel().equals("UnfilteredField")));
     }
 
+    @Test
+    public void shouldGetRegionUUIDForCase() {
+
+        Correspondent correspondent = new Correspondent(UUID.randomUUID(), "TYPE", "name",
+                new Address("postcode","address1","address2","address3","county"),
+                "phone", "email", "", "extKey");
+        //CaseData caseData = new CaseData(caseType, caseID, new HashMap<>(), objectMapper, caseReceived);
+        CaseData caseData = mock(CaseData.class);
+        when(caseData.getPrimaryCorrespondent()).thenReturn(correspondent);
+        when(caseDataRepository.findByUuid(caseData.getUuid())).thenReturn(caseData);
+        ConstituencyDto constituencyDto = mock(ConstituencyDto.class);
+        when(constituencyDto.getRegionUUID()).thenReturn(UUID.fromString("85462836-4de7-4297-b60f-2bd260b3a686"));
+        when(infoClient.getConstituencyByMemberExternalKey("extKey")).thenReturn(constituencyDto);
+        UUID regionUUID = caseDataService.getRegionUUIDForCase(caseData.getUuid());
+
+        verify(caseDataRepository, times(1)).findByUuid(caseData.getUuid());
```

Comment:
'times(1)' is the default value anyway so we can safely remove it from here:

verify(caseDataRepository).findByUuid(caseData.getUuid());

Reply:
Safely removed, as requested. (Not retro-applied to all other usages.)